### DISCUSSION TITLE
cloudflare us.kg这样的顶级域名报错

### DISCUSSION THREAD

**Discussion by weilining** [jeessy2/ddns-go#1233] [Q&A]
2024/08/27 21:04:04 你的IP 2409:8a9c:1c69:dfb5:2e0:b4ff:fe1b:628 没有变化, 域名 ipv6.nas.333349.xyz
2024/08/27 21:04:04 在DNS服务商中未找到根域名: wning.us.kg

设置如下 ipv6:wning.us.kg

**Comment by TerryZ** [comment]
Same error

**Comment by jeessy2** [comment]
换个正常点的域名
